src/front/app/features/main/profile/components/UserDetails/Stat.js

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react';
+
+const Stat = ({ title, value, icon }) => (
+    <li className="user__stats__item" >
+        <div className="user__stats__item__value" >
```

Comment:
actually I don`t touch this component... Saif wrote this, it doesn`t even used in UserDetails but I left it cause maybe it will be useful for styling, so if it is maybe it`s better to refactor it on UI task or if this component not useful for styling we can delete it)